frankapy_control_test_scripts/README.md

```diff
@@ -0,0 +1,51 @@
+- goto_pose 
+    - if dynamic is set as True: 
+        - This mode allows us to stream joint commands until the duration period passed to goto_pose is done.
+        - custom cartesian impedance controller is used (they claim variable impedance)
+    - if dynamic is set as False
+        - This is like a setpoint controller, can't stream desired pose. 
+        - if use_impedance is True: 
+            - custom cartesian impedance controller is used (they claim variable impedance)
+        - is use_impedance is False: 
+            - libfranka's internal IK + joint impedance controller is used     
+
+- franka is precise for large magnitude motions but imprecise for smaller motions  
+
+### [docker_frankapy_test.py](docker_frankapy_test.py):
+
+This script can be used to test to see if both the realtime and workstation docker installations work without any errors along with running frankapy. Expected output: the robot arm "resets joints" to go to the home configuration. 
+
+### [test_ee_pose_control.py](test_ee_pose_control.py):
+
+This script uses dynamic pose control mode of frankapy to stream a square on the YZ plane (in robot base frame). We also set higher stiffness values than default values to get accurate pose tracking. 
+
+In lines 20-30 you can specify the edge length of the square trajectory, speed at which we want the end-effector to trace one edge of the square, delta time steps at which we sample the desired square trajectory and the no.of cycles of tracing the square trajectory, rotational and translational stiffness of the cartesian impedance control (roll, pitch, yaw of the square in the robot base frame, but its not implemented right now). After executing the input trajectory the commanded and actual end-effector positions are plotted. 
+
+### [test_ee_pose_impedance_control.py](test_ee_pose_impedance_control.py):
+
+This script uses dynamic pose control mode of frankapy to stream a square on the YZ plane (in robot base frame). We also test variable impedance capability of the controller. 
+
+In lines 20-30 you can specify the edge length of the square trajectory, speed at which we want the end-effector to trace one edge of the square, delta time steps at which we sample the desired square trajectory and the no.of cycles of tracing the square trajectory, rotational and translational stiffness of the cartesian impedance control (roll, pitch, yaw of the square in the robot base frame, but its not implemented right now). After executing the input trajectory the commanded and actual end-effector positions are plotted. We also plot the translational impedance value that we have commanded. 
+
+### [test_joint_position_control.py]([test_joint_position_control.py]):
+
+This script tests uses the dynamic joint control mode of frankapy to stream sinusoid reference trajectories to joints 1,4,5,6 of the franka robot arm. After executing the input trajectory the commanded and actual joint angles are plotted. 
+
```

Comment:
maybe warn the user stay away from the robot. if you want to add a figure of the robot at the highest height, might give the use the impression how the robot will move.